src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.Print.cs

```diff
@@ -25,16 +28,36 @@ internal static unsafe string DerStringToManagedString(byte[] derString)
                 return null;
             }
 
+            using (asn1String)
+            {
+                return Asn1StringToManagedString(
+                    asn1String,
+                    (bio, str, flags) => Asn1StringPrintEx(bio, str, flags));
```

Comment:
I think you can just make this:

```C#
return Asn1StringToManagedString(  
  asn1String,  
  Asn1StringPrintEx);  
```
And it will save on the creation of a new anonymous inline method.

Same comment for the method below.